pages/dkp/kommander/2.3/clusters/custom-domain-certificate/managed/index.md

```diff
@@ -0,0 +1,146 @@
+
+This section describes how to enable custom domains and certificates in your **managed and attached** clusters. Refer to the [Customize a domain and certificate in the Management Cluster][management] documentation, if you want to set up custom domains and certificates for your **Management Cluster** during the installation of DKP.
+
+To customize the domain or certificate for a specific cluster after the installation of DKP, adapt or create an API `yaml` that will allow Kommander to implement the established adjustments on top of the default or any previous configuration you have made.
+
+## Configure custom domains or custom certificates
+
+<p class="message--warning"><strong>IMPORTANT:</strong>Ensure your <code>dkp</code> configuration references the management cluster of the environment where you want to customize the domain or certificate by setting the <code>KUBECONFIG=<path></code> environment variable, or using the <code>--kubeconfig</code> flag, <a href="https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/">in accordance with Kubernetes conventions</a>.
+
+Use the API yaml to customize the domain (via the `hostname` field), the certificate (via the `issuerRef` or `certificateSecretRef` field), or both. For this, refer to the following examples:
+
+1.  Create or update and apply the `KommanderCluster` object with the wanted ingress. Remember to specify the cluster in the `kubeconfigRef` name field.
+
+    In this example, you can enter the custom domain in the `hostname` field, and an **issuer to be used by cert-manager** to issue a certificate for the domain in the `issuerRef` field.
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: kommander.mesosphere.io/v1beta1
+    kind: KommanderCluster
+    metadata:
+      name: <cluster_name>
+      namespace: <workspace_namespace>
+    spec:
+      kubeconfigRef:
+        name: <cluster_name>-kubeconfig
+      clusterRef:
+        capiCluster:
+          name: <cluster_name>
+    ingress:
+      hostname: <cluster_hostname>
+      issuerRef:
+        namespace: <issuer_namespace>
+        name: <issuer_name>
+    EOF
+    ```
+
+    In this example, you can enter the custom domain in the `hostname` field, and the secret in the `certificateSecretRef` field for **customized certificates created for your hostname**:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: kommander.mesosphere.io/v1beta1
+    kind: KommanderCluster
+    metadata:
+      name: <cluster_name>
+      namespace: <workspace_namespace>
+    spec:
+      kubeconfigRef:
+        name: <cluster_name>-kubeconfig
+      clusterRef:
+        capiCluster:
+          name: <cluster_name>
+    ingress:
+      hostname: <cluster_hostname>
+      certificateSecretRef:
+        namespace: <secret_namespace>
+        name: <secret_name>
+    EOF
+    ```
+
+<!-- 1.  TODO: Dev - Are there other steps, is there some sort of confirmation message or output?  -->
```

Comment:
@mhrabovcin @floridoo we can delete this line (69) if there are no further steps here. But I guess it would be nice to have some confirmation output example here if there is one?

Reply:
Success / Errors can be inspected in the `status.conditions` section of the `KommanderCluster` object. That's the same as you describe in the "TroubleShooting" section. Maybe we could rename that one to something reflecting better that what's written there also applies to successful setups.

Reply:
Added some examples https://github.com/mesosphere/dcos-docs-site/pull/4553/files#r905853722

Reply:
Addressed in https://github.com/mesosphere/dcos-docs-site/pull/4553/commits/e4c6941f6078771b4a35480f70238368e049bad7